### PR TITLE
Add command `brctl` to examples.

### PR DESCRIPTION
**Pull request by Lirt**
Signed-off-by: Ondrej Vasko <ondrej.vasko@vnet.eu>

### CODE REVIEW

**Comment by Lirt** [comment]
Please redo travis test. There was some internal error in travis.

File: eg/examples/brctl.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    brctl stp br1 on
+
+Setup ageing of MAC addresses in bridge `br1` to 0 seconds. This will cause bridge to not learn MAC addresses:
+
+    brctl setageing br1 0
+
+Show details about bridge `br1`:
+
+    brctl showstp `br1`
```
Comment (srsudar) [inline eg/examples/brctl.md]: I think this should be `brctlshowstp br1`, without the backticks. Do you agree?

Comment (Lirt) [reply eg/examples/brctl.md]: Yes I think I left it by mistake.

**Comment by srsudar** [comment]
Great, thanks. Sorry this has taken me some time to get to. `brctl` on my Mac is a document cloud server, but I just tried most of the commands on a Linux machine and they work great.

This might be the first tool that has a name conflict on Mac/Linux and isn't just a few subtle API differences. If someone submits the other one I'm going to have to think about how to support different systems, eg `man 8 brctl`, which I've been punting on.

I'm going to make a few edits to bring it in line with my draconian spacing requirements and then will include it in a release, hopefully within a few days.

**Comment by Lirt** [comment]
That's interesting, I didn't know about it. 

I got no idea about how to make interface to it but thank you for merge.

**Pull request merged**